### PR TITLE
feat: add respec build script

### PR DESCRIPTION
**Pull request by sublimator**

### CODE REVIEW

**Comment by sublimator** [comment]
@huijing

**Comment by sublimator** [comment]
@huijing 

Feel free to fix this up to your liking!

I was updating the `retired` date and noticed I needed to "build" the specification.html and then got carried away with the includes. It's something I'd talked to @sidvishnoi (respec maintainer) about before. 

Thanks.

**Comment by sublimator** [comment]
@huijing 

Bump. I can make a PR with just the `retired` changes if you'd like

**Review by huijing — APPROVED** [review]
@sublimator so sorry for missing this. it looks good to me. and the deploy preview works as well. i noticed u cleaned up the example code as well, so thank you for that.

**Comment by sublimator** [comment]
Thanks!

File: .github/workflows/respec.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  push:
+    paths:
+      - 'src/pages/specification/includes'
+      - 'src/pages/specification/specification-respec.html'
+    branches:
+      - gh-pages
+
+jobs:
+  tidy:
+    name: Rebuild Static
```
Comment (sublimator) [inline .github/workflows/respec.yml:13]: not sure if this will work - but sure you all can fix it

**Pull request merged**